spec/features/user_deletes_book_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature 'User deletes a book', js: true do
+  scenario 'successfully' do
+    book = create(:book)
+    visit root_path
+
+    delete_book
+
+    expect(page).to have_content(t('books.index.header'))
+    expect(page).not_to have_content(book.title)
+    expect(page).not_to have_content(book.author)
+    expect(page.current_path).to eq root_path
```

Comment:
You probably don't need this since the test will fail if the books index header isn't there